docs/encyclopedia/nexus-operations.mdx

```diff
@@ -167,6 +170,78 @@ This differs from how Activities and Workflows handle errors and retries:
 
 See [errors in Nexus handlers](/nexus/error-handling#errors-in-nexus-handlers) to control the retry behavior by returning a [non-retryable Nexus error](/references/failures#non-retryable-nexus-errors).
 
+## Circuit breaking {#circuit-breaking}
+
+Nexus provides a circuit breaker for each destination pair (caller Namespace and target Nexus Endpoint). The circuit breaker for each pair trips and resets independently.
+By default, the circuit breaker kicks in after 5 consecutive Nexus requests fail with a [retryable error](/references/failures#nexus-errors).
+For example, this happens if all Nexus Workers for a Nexus Endpoint are down and 5 consecutive requests fail due to request timeouts.
+
+Once a circuit breaker has tripped and is in the open state, the caller's Nexus Machinery will fail early and won't send requests to the target Nexus Endpoint.
+After 60 seconds in open state, it will change to the half-open state, which will allow only 1 request to be made.
+If the request is successful, then the circuit breaker changes its state to closed, and allows all requests to pass through.
+
+The circuit breaker state is surfaced in a caller Workflow's [Pending Nexus Operations](/nexus/execution-debugging#pending-operations) and [Pending Nexus Callbacks](/nexus/execution-debugging#pending-callbacks).
```

Comment:
Can you be specific here that "Pending Nexus Callbacks" is on the handler side?
The way it's written sounds like the callbacks are also in the caller workflow.

```suggestion
The circuit breaker state is surfaced in a caller Workflow's [Pending Nexus Operations](/nexus/execution-debugging#pending-operations), and in the handler's Workflow [Pending Nexus Callbacks](/nexus/execution-debugging#pending-callbacks).
```